pipeline_dp/aggregate_params.py

```diff
@@ -104,7 +104,7 @@ class SumParams:
             partition.
         public_partitions: A collection of partition keys that will be present in
             the result.
-        partition_extractor: A function for partition id extraction from an RDD record.
+        partition_extractor: A function for partition id extraction from an collection record.
```

Comment:
an collection -> a collection (and in other places)

Reply:
Thanks! Done.